mobile-center-link-scripts/src/ios/MobileCenterConfig.js

```diff
@@ -40,7 +40,15 @@ function addConfigToProject(file) {
     return new Promise(function (resolve, reject) {
         debug(`Trying to add ${file} to XCode project`);
 
-        var globString = `**/${(pjson && pjson.name ? pjson.name : '*')}.xcodeproj/project.pbxproj`
+        var globString;
+        if (pjson && pjson.name) {
+            globString = `**/${(pjson.name.replace(/-/g, ''))}`;
+        }
+        else {
+            globString = `ios/*`;
+        }
+        globString += '.xcodeproj/project.pbxproj';
+
         var projectPaths = glob.sync(globString, { ignore: 'node_modules/**' });
```

Comment:
@max-mironov I gave the repro in https://github.com/Microsoft/mobile-center-sdk-react-native/pull/51#issue-244533783 it's when using create-react-native app then eject.

Also the reason to use ios folder in the failover is that otherwise with ** it also matches the pods xcode project in the sub directory and thus fail.